src/components/Icons/FiatIcon.js

```diff
@@ -19,34 +19,34 @@
 */
 
 import * as React from 'react';
-
-// Components
-import Image from 'components/Image';
-
-// Utils
-import { useThemedImages } from 'utils/images';
+import { Image } from 'react-native';
 
 // Types
-import type { ViewStyleProp } from 'utils/types/react-native';
+import type { ImageStyleProp } from 'utils/types/react-native';
+import type { Currency } from 'models/Rates';
+
+// Assets
+const iconUSD = require('assets/icons/icon-48-fiat-usd.png');
+const iconEUR = require('assets/icons/icon-48-fiat-eur.png');
+const iconGBP = require('assets/icons/icon-48-fiat-gbp.png');
 
 type Props = {|
-  url: ?string,
-  diameter?: number,
-  style?: ViewStyleProp
+  currency: Currency,
+  size?: number,
+  style?: ImageStyleProp,
 |};
 
-function TokenIcon({ url, diameter = 48, style }: Props) {
-  const { genericToken } = useThemedImages();
+function FiatIcon({ currency, size = 48, style }: Props) {
+  const source = iconSoureForCurrency[currency];
```

Comment:
Seems like `FiatIcon` would no longer have `genericToken` fallback, is this intentional?

Reply:
I'll add fallback for fiat as well